charts/spark-operator-chart/crds/sparkoperator.k8s.io_scheduledsparkapplications.yaml

```diff
@@ -18,4479 +17,11598 @@ spec:
+                description: ConcurrencyPolicy is the policy governing concurrent
+                  SparkApplication runs.
+                type: string
+              failedRunHistoryLimit:
+                description: |-
+                  FailedRunHistoryLimit is the number of past failed runs of the application to keep.
+                  Defaults to 1.
+                format: int32
+                type: integer
+              schedule:
+                description: Schedule is a cron schedule on which the application
+                  should run.
+                type: string
+              successfulRunHistoryLimit:
+                description: |-
+                  SuccessfulRunHistoryLimit is the number of past successful runs of the application to keep.
+                  Defaults to 1.
+                format: int32
+                type: integer
+              suspend:
+                description: |-
+                  Suspend is a flag telling the controller to suspend subsequent runs of the application if set to true.
+                  Defaults to false.
+                type: boolean
+              template:
+                description: Template is a template from which SparkApplication instances
+                  can be created.
+                properties:
+                  arguments:
+                    description: Arguments is a list of arguments to be passed to
+                      the application.
+                    items:
                       type: string
-                    batchSchedulerOptions:
-                      properties:
-                        priorityClassName:
+                    type: array
+                  batchScheduler:
+                    description: BatchScheduler configures which batch scheduler will
+                      be used for scheduling
+                    type: string
+                  batchSchedulerOptions:
+                    description: BatchSchedulerOptions provides fine-grained control
+                      on how to batch scheduling.
+                    properties:
+                      priorityClassName:
+                        description: PriorityClassName stands for the name of k8s
+                          PriorityClass resource, it's being used in Volcano batch
+                          scheduler.
+                        type: string
+                      queue:
+                        description: Queue stands for the resource queue which the
+                          application belongs to, it's being used in Volcano batch
+                          scheduler.
+                        type: string
+                      resources:
+                        additionalProperties:
+                          anyOf:
+                          - type: integer
+                          - type: string
+                          pattern: ^(\+|-)?(([0-9]+(\.[0-9]*)?)|(\.[0-9]+))(([KMGTPE]i)|[numkMGTPE]|([eE](\+|-)?(([0-9]+(\.[0-9]*)?)|(\.[0-9]+))))?$
```

Comment:
Can we, at minimum, comment this to make it more clear what we're trying to match?